theano/tensor/basic.py

```diff
@@ -1427,21 +1427,31 @@ class Argmax(Op):
     nin = 2  # tensor, axis
     nout = 1
     E_axis = 'invalid axis'
-    __props__ = ()
+    __props__ = ('axis',)
```

Comment:
It can be in the non-optimized graph... But the probability is pretty low. I'll let @lamblin choose.